openfisca_aotearoa/variables/social_security/child_disability_allowance.py

```diff
@@ -0,0 +1,70 @@
+from openfisca_aotearoa.entities import Person, Family
+from openfisca_core.model_api import *
+
+
+class social_security__eligible_for_child_disability_allowance(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = "Eligible for Child Disability Allowance"
+
+    def formula(persons, period, parameters):
+        # The applicant
+        in_nz = persons('normally_lives_in_nz', period)
+        resident_or_citizen = persons('is_resident', period) + \
+            persons('is_permanent_resident', period) + \
+            persons('is_nz_citizen', period)
+
+        is_principal_carer = persons.has_role(Family.PRINCIPAL_CAREGIVER)
+
+        has_disabled_child = persons.family("social_security__family_has_disabled_child", period)
+
+        return in_nz * \
+            resident_or_citizen * \
+            is_principal_carer * \
+            has_disabled_child
+
+
+class social_security__family_has_disabled_child(Variable):
+    value_type = bool
+    entity = Family
+    definition_period = MONTH
+    label = u'Does the family have a child who meets the criteria for disabled'
```

Comment:
Unicode literals are probably not required?

Reply:
We've gone with adding them before the strings that appear to the human users. It's a convention we learned from France's implementation.